src/pages/assignments/components/assignment-card.vue

```diff
@@ -214,13 +224,57 @@ q-card.assignment(v-if="isFiltered && ((isExpired && history) || (!isExpired &&
         )
           q-item-section(style="max-width: 20px;")
             q-icon(name="fas fa-pencil-alt" size="14px")
-          q-item-section Edit
+          q-item-section Modify
+        q-item(
+          v-if="account === assignee"
+          clickable
+        )
+          q-popup-proxy(@before-show="newCommit = commited")
+            .confirm.column.q-pa-sm(style="max-width: 275px; background:white;")
+              | If you adjust your assignment, your % commitment will be immediately
+              | reflected on your next claim (no vote is necessary). Multiple adjustments
+              | during the same claim period will be included in the calculation.
+              .commit-group.q-mt-sm.q-px-sm.q-pb-sm(style="background: #EFEFEF;")
```

Comment:
You create a new css class but not use it and add inline CSS
I would use the css class you created